frontend/src/metabase/new_query/containers/NewQueryOptions.styled.tsx

```diff
@@ -24,6 +27,20 @@ export const QueryOptionsRoot = styled.div`
   }
 `;
 
+export const EducationalButton = styled(ExternalLink)<EducationalButtonProps>`
```

Comment:
`EducationalButtonProps` seems to be not needed here

Reply:
Got it, thanks!